Correct a typo in the Unittest documentation (#10397)
python/cpython: Doc/library/unittest.rst

.. versionadded:: 3.4

> When some of your tests differ only by some very small differences, for

Reviewer: Thanks for your PR, but I think that "some very small differences" is sounding odd 😕 

What about

```suggestion
When there are very small differences among your tests, for
```
